src/routes/_utils/blurhash.js

```diff
@@ -0,0 +1,23 @@
+import { decode as decodeBlurHash } from 'blurhash'
+import { mark, stop } from './marks'
+
+const RESOLUTION = 32
```

Comment:
We could maybe lower this based on some heuristics (e.g. `navigator.hardwareConcurrency`)? Setting it to 1 will basically extract the average color, so maybe we can do that initially and repeat with 32 inside a `requestIdleCallback`?

Reply:
I'm not sure what the limiting factor is for blurhash, but yeah for now a hard-coded value is probably fine.